docs/sources/flow/reference/components/loki.source.file.md

```diff
@@ -107,4 +133,59 @@ loki.write "local" {
 }
 ```
 
+### File globbing
```

Comment:
Based on questions from users, I found that it is useful to add this as an example too.